tests/network/primary_pod_network.go

```diff
@@ -82,6 +86,59 @@ var _ = SIGDescribe("Primary Pod Network", func() {
 			It("should report PodIP as its own on interface status", func() { AssertReportedIP(vmi) })
 		})
 	})
+
+	Describe("Connectivity", func() {
+		const servicePort = 1500
+
+		AssertConnectivity := func(outboundVMI, inboundVMI *v1.VirtualMachineInstance) {
+			inboundVMIAddress := inboundVMI.Status.Interfaces[0].IP
+
+			By("Checking connectivity via ICMP")
+			Expect(tests.PingFromVMConsole(outboundVMI, inboundVMIAddress)).To(Succeed())
+
+			By("Checking connectivity via ICMP with MTU-sized frames")
+			mtu := getPodNetworkMTU(outboundVMI)
+			ipHeaderSize := 28 // IPv4 specific
+			Expect(tests.PingFromVMConsole(outboundVMI, inboundVMIAddress, "-c1", "-w5", "-s", strconv.Itoa(mtu-ipHeaderSize))).To(Succeed())
```

Comment:
I think you should send few more packets to verify there are no drops

Reply:
Done